R/post_hit_AUC.R

```diff
@@ -0,0 +1,48 @@
+#' Calculate AUC After Hit-calling
+#'
+#' Function that calculates AUC for dose-response curves after hit-calling
+#'
+#' This function calculates AUC value for the winning model selected after
+#' hit-calling.
+#'
+#'
+#' @param hit_results output from tcplhit2_core
+#'
+#' @return AUC value of the winning model
+#'
+#' @export
+#' @importFrom stringr str_split
+#'
+#' @examples
+#'
+#' conc <- list(.03, .1, .3, 1, 3, 10, 30, 100)
```

Comment:
Make sure you update this example since post_hit_AUC works within the concRespCore function.